test/OpenTelemetry.Exporter.Geneva.Benchmarks/Exporter/TraceExporterBenchmarks.cs

```diff
@@ -99,6 +150,24 @@ public void SerializeActivity()
         this.exporter.SerializeActivity(this.activity);
     }
 
+    [Benchmark]
+    public void SerializeActivityWithoutTraceState()
```

Comment:
`TraceState` is too specific to have dedicated benchmarks for it. I would be in favor of keeping the benchmarks simple and targeted towards the primary use case.

Reply:
we only need it for discussion in this PR, no need to merge this changes.

Reply:
Sure, I'm temporarily using it to test the impact. I'll remove it from the Benchmark.

Reply:
Reverted all changes in Benchmark.